tests/integ/modin/series/test_str_accessor.py

```diff
@@ -221,9 +221,10 @@ def test_str_replace_neg(pat, n, repl, error):
         snow_ser.str.replace(pat=pat, repl=repl, n=n)
 
 
+@pytest.mark.xfail(reason="Series.str.split() is not implemented", strict=True)
```

Comment:
I don't think this should be an XFAIL, we should have `pytest.raises(NotImplementedError)` instead.